### DISCUSSION TITLE
How do I add the logger LWC from the managed package?

### DISCUSSION THREAD

**Discussion by mikesobczak** [jongpie/NebulaLogger#440] [General]
I'm using the 4.9 managed package.  I'm trying to include the logger LWC in my component, but none of the following work:

```
<nebula-logger></nebula-logger>
<Nebula-logger></Nebula-logger>
```

They both return **No MODULE named Nebula:logger found** when I try to deploy my custom component.  What am I doing wrong?

**Comment by KumaranM2040** [comment]
Have a look at https://github.com/jongpie/NebulaLogger/blob/main/nebula-logger/recipes/lwc/loggerLWCDemo/loggerLWCDemo.html

You need add this to your component:
<c-logger></c-logger>
